Exception in thread "main" Fatal error:
Unknown exception occurred: java.lang.RuntimeException: java.lang.IllegalArgumentException: No such atom in the universe: 1024
	at edu.mit.csail.sdg.alloy4compiler.translator.TranslateAlloyToKodkod.execute_command(Unknown Source)
	at edu.mit.csail.sdg.alloy4whole.multiobjective.RanMultiobjectiveModel.main(Unknown Source)
Caused by: java.lang.RuntimeException: java.lang.IllegalArgumentException: No such atom in the universe: 1024
	at kodkod.multiobjective.concurrency.BlockingSolutionIterator.hasNext(BlockingSolutionIterator.java:54)
	at edu.mit.csail.sdg.alloy4compiler.translator.A4Solution$Peeker.<init>(Unknown Source)
	at edu.mit.csail.sdg.alloy4compiler.translator.A4Solution$Peeker.<init>(Unknown Source)
	at edu.mit.csail.sdg.alloy4compiler.translator.A4Solution.solve(Unknown Source)
	... 2 more
Caused by: java.lang.IllegalArgumentException: No such atom in the universe: 1024
	at kodkod.instance.Universe.index(Universe.java:168)
	at kodkod.instance.TupleFactory$IntTuple.<init>(TupleFactory.java:328)
	at kodkod.instance.TupleFactory.tuple(TupleFactory.java:74)
	at kodkod.intelision.IntExprReduction.recreateUniverseAndBounds(IntExprReduction.java:394)
	at kodkod.engine.CheckpointedSolver.solve(CheckpointedSolver.java:137)
	at kodkod.multiobjective.algorithms.CheckpointedGuidedImprovementAlgorithm.multiObjectiveSolveImpl(CheckpointedGuidedImprovementAlgorithm.java:45)
	at kodkod.multiobjective.algorithms.MultiObjectiveAlgorithm.multiObjectiveSolve(MultiObjectiveAlgorithm.java:46)
	at kodkod.multiobjective.MultiObjectiveSolver$1.run(MultiObjectiveSolver.java:84)
	at java.lang.Thread.run(Thread.java:722)
